packages/snap-preact/components/src/components/Organisms/Facet/Facet.tsx

```diff
@@ -393,15 +417,11 @@ const FacetContent = (props: any) => {
 						cloneWithProps(overflowSlot, { facet })
 					) : (
 						<Fragment>
-							{
-								<Icon
-									{...subProps.showMoreLessIcon}
-									{...(((facet as ValueFacet).overflow?.remaining || 0) > 0
-										? { ...(typeof iconOverflowMore == 'string' ? { icon: iconOverflowMore } : (iconOverflowMore as Partial<IconProps>)) }
-										: { ...(typeof iconOverflowLess == 'string' ? { icon: iconOverflowLess } : (iconOverflowLess as Partial<IconProps>)) })}
-								/>
-							}
-							<span>{((facet as ValueFacet)?.overflow?.remaining || 0) > 0 ? showMoreText : showLessText}</span>
+							<Icon
+								{...subProps.showMoreLessIcon}
+								icon={((facet as ValueFacet).overflow?.remaining || 0) > 0 ? iconOverflowMore : iconOverflowLess}
```

Comment:
Change removes support for using IconProps in `iconOverflowMore` and `iconOverflowLess`.